api/app/views/spree/api/variants/small.v1.rabl

```diff
@@ -14,4 +14,4 @@ child :option_values => :option_values do
   attributes *option_value_attributes
 end
 
-child(:images => :images) { extends "spree/api/images/show" }
+child(:display_images => :images) { extends "spree/api/images/show" }
```

Comment:
I tested searching for variants when creating an order and a stock transfer in admin with a large dataset. I didn't notice any performance issues there but let me know if there's any other scenario you or @athal7 are particularly concerned about that I'm missing.